app/billing_client.go

```diff
@@ -0,0 +1,310 @@
+	BillingEmail string
+	// defined if type is PaymentMethodTypeCard
+	Method *PaymentMethodCard
+	// only return for billing dashboard admin users
+	BillingTier *string
+}
+
+func getOrgBillingInformationResponseFromProto(resp *pb.GetOrgBillingInformationResponse) *GetOrgBillingInformationResponse {
+	return &GetOrgBillingInformationResponse{
+		Type:         paymentMethodTypeFromProto(resp.Type),
+		BillingEmail: resp.BillingEmail,
+		Method:       paymentMethodCardFromProto(resp.Method),
+		BillingTier:  resp.BillingTier,
+	}
+}
+
+// InvoiceSummary holds the information of an invoice summary.
+type InvoiceSummary struct {
+	ID            string
+	InvoiceDate   *timestamppb.Timestamp
+	InvoiceAmount float64
+	Status        string
+	DueDate       *timestamppb.Timestamp
+	PaidDate      *timestamppb.Timestamp
+}
+
+func invoiceSummaryFromProto(summary *pb.InvoiceSummary) *InvoiceSummary {
+	return &InvoiceSummary{
+		ID:            summary.Id,
+		InvoiceDate:   summary.InvoiceDate,
+		InvoiceAmount: summary.InvoiceAmount,
+		Status:        summary.Status,
+		DueDate:       summary.DueDate,
+		PaidDate:      summary.PaidDate,
+	}
+}
+
+// BillingClient is a gRPC client for method calls to the Billing API.
+type BillingClient struct {
+	client pb.BillingServiceClient
+}
+
+// NewBillingClient constructs a new BillingClient using the connection passed in by the Viam client.
+func NewBillingClient(conn rpc.ClientConn) *BillingClient {
+	return &BillingClient{client: pb.NewBillingServiceClient(conn)}
+}
+
+// GetCurrentMonthUsage gets the data usage information for the current month for an organization.
+func (c *BillingClient) GetCurrentMonthUsage(ctx context.Context, orgID string) (*GetCurrentMonthUsageResponse, error) {
+	resp, err := c.client.GetCurrentMonthUsage(ctx, &pb.GetCurrentMonthUsageRequest{
+		OrgId: orgID,
+	})
+	if err != nil {
+		return nil, err
+	}
+	return getCurrentMonthUsageResponseFromProto(resp), nil
+}
+
+// GetOrgBillingInformation gets the billing information of an organization.
+func (c *BillingClient) GetOrgBillingInformation(ctx context.Context, orgID string) (*GetOrgBillingInformationResponse, error) {
+	resp, err := c.client.GetOrgBillingInformation(ctx, &pb.GetOrgBillingInformationRequest{
+		OrgId: orgID,
+	})
+	if err != nil {
+		return nil, err
+	}
+	return getOrgBillingInformationResponseFromProto(resp), nil
+}
+
+// GetInvoicesSummary returns the outstanding balance and the invoice summaries of an organization.
+func (c *BillingClient) GetInvoicesSummary(ctx context.Context, orgID string) (float64, []*InvoiceSummary, error) {
+	resp, err := c.client.GetInvoicesSummary(ctx, &pb.GetInvoicesSummaryRequest{
+		OrgId: orgID,
+	})
+	if err != nil {
+		return 0, nil, err
+	}
+	var invoices []*InvoiceSummary
+	for _, invoice := range resp.Invoices {
+		invoices = append(invoices, invoiceSummaryFromProto(invoice))
+	}
+	return resp.OutstandingBalance, invoices, nil
+}
+
+// GetInvoicePDF gets the invoice PDF data.
```

Comment:
A small elaboration here about the returned byte slice representing the raw bytes of the obtained PDF file would be great.